tests/mesh/test-graph.cpp

```diff
@@ -120,13 +121,19 @@ void test_delete_edge( const geode::Graph& graph, geode::GraphBuilder& builder )
         "[Test] Graph edge vertex index is not correct" );
 
     builder.create_edges( 10 );
+    builder.set_edge_vertex( { 1, 0 }, 1 );
+    builder.set_edge_vertex( { 1, 1 }, 0 );
     OPENGEODE_EXCEPTION(
         graph.nb_edges() == 11, "[Test] Graph should have 11 edges" );
 
     to_delete.resize( graph.nb_edges(), true );
     builder.delete_edges( to_delete );
     OPENGEODE_EXCEPTION(
         graph.nb_edges() == 1, "[Test] Graph should have 1 edge" );
+    OPENGEODE_EXCEPTION( graph.edge_vertex( { 0, 0 } ) == 1,
+        "[Test] Graph edge vertex index is not correct" );
+    OPENGEODE_EXCEPTION( graph.edge_vertex( { 0, 1 } ) == 0,
+        "[Test] Graph edge vertex index is not correct" );
```

Comment:
```suggestion
        "[Test] Graph edge vertex index is not correct (0,1)" );
```
The objective is to have different error messages to quickly differentiate that is wrong